preload/prefetch-cache.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
I wonder if all these files should go in https://github.com/web-platform-tests/wpt/tree/master/html/semantics/links/linktypes instead? Not sure, and I guess it's OK either way...